packages/cypress-commands/src/setups/enableNetworkShim/index.js

```diff
@@ -44,25 +44,11 @@ export function enableNetworkShim() {
             // First get the updated local state from the alias
             cy.get('@networkShimState').then(networkShimState => {
                 /*
-                 * In capture mode the state needs to be incrementally updated
-                 * across tests, so after every feature the entire plugin state
+                 * In both capture and stub mode the state needs to be incrementally 
+                 * updated across tests, so after every feature the entire plugin state
                  * gets overwritten.
                  */
-                if (isCaptureMode()) {
-                    cy.task('setNetworkShimState', networkShimState)
-                }
-                /*
-                 * In stub mode the state needs to be kept static across features
-                 * apart from the missing request stubs which do need to be
-                 * incrementally updated across features. So in stub mode we only
-                 * update that state property in the plugin.
-                 */
-                if (isStubMode()) {
-                    cy.task(
-                        'setNetworkShimMissingRequestStubs',
-                        networkShimState.missingRequestStubs
-                    )
-                }
```

Comment:
After fixing how fixtures were stored in `captureRequest`, stub-mode was still broken. It turned out this was because of the fact that the `responseCount` property of a `requestStub` was being cleared between tests. In order to preserve the value of `responseCount` between tests, we need to handle the state in exactly the same way as we do in capture mode so that it is incrementally updated between tests.